app/src/androidTest/java/org/mozilla/focus/activity/TestHelper.java

```diff
@@ -234,6 +235,14 @@ static void openNotification() {
         mDevice.openNotification();
     }
 
+    static void clearAllNotification() throws UiObjectNotFoundException {
+        UiObject clearButton = mDevice.findObject(new UiSelector()
+                .resourceId("com.android.systemui:id/dismiss_text")
+                .enabled(true));
+        clearButton.click();
+        clearButton.waitUntilGone(waitingTime);
+    }
```

Comment:
I don't see this new method being used in this patch?

Reply:
yes, I thought this can help our notification issue, then in some sim, where there were multiple notifications, this 'clear all' button was not visible.  I left it but since I'm not using it, perhaps I should remove it.